src/corehost/cli/fxr/fx_resolver.cpp

```diff
@@ -286,7 +320,7 @@ StatusCode fx_resolver_t::soft_roll_forward_helper(
 //  - fx_ref
 //      The reference to resolve (the one we're processing).
 //      Passed by-value to avoid side-effects with mutable newest_references and oldest_references.
-//  - newest_is_hard_roll_forward
+//  - fx_is_hard_resolved
 //      true if there is a reference to the framework specified by fx_ref in the newest_references
```

Comment:
This line is no longer true.

Reply:
I don't see why not - AFAIK it does correctly describe the code behavior.